src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+@import "../utils/variables.scss";
+
+.card {
+  padding: 32px 16px 16px;
+  font-size: 12px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 auto;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-color;
+    margin-bottom: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    margin-bottom: 16px;
+    color: $gray-color;
+    margin-top: 4px;
+  }
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews-number {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $gray-color;
+    }
+
+    &-numbers {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $primary-color;
+    }
+  }
+
+  &__button {
+    background-color: $btn-background;
```

Comment:
check the correct order of CSS props: [link](https://9elements.com/css-rule-order/), [example](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/)